dcpy/models/file.py

```diff
@@ -2,8 +2,10 @@
 from pydantic import BaseModel
 from typing import Literal, TypeAlias
 
+from dcpy.models.geospatial import geometry
 
-class Geometry(BaseModel):
+
+class Geometry(BaseModel, extra="forbid"):
```

Comment:
not urgent at all, but curious what you think. Should we move this class to `models/geospatial/geometry.py`? Seems like it belongs there along with `GeometryFormat`

Reply:
Yeah I was wondering a bit about this. It is fairly specific to tabular geometry - a set of columns denoting a definition of geometry. So still maybe, but I might want to call if something slightly more specific that just `Geometry`. But I don't quite know what that name is (a classic tale)

Reply:
But it certainly belongs more over there than here